projects/plugins/jetpack/_inc/client/main.jsx

```diff
@@ -701,6 +715,9 @@ class Main extends React.Component {
 						message={ this.handleRouterWillLeave }
 					/>
 					{ this.renderMainContent( this.props.location.pathname ) }
+					{ this.shouldShowAgenciesCard() && (
+						<AgenciesCard path={ this.props.location.pathname } />
```

Comment:
I'm wondering if path could be read from inside the `AgenciesCard` component.

Reply:
You mean like just as `window.location.pathname`? I guess I wasn't sure if the `props.location.pathname` was different in any way and saw other components using this pattern so I went with it. However if it is the same thing, I could see that being better

Reply:
Looks like when I use `location.pathname` I get `/wp-admin/admin.php` but when I use `this.props.location.pathname` I get `/dashboard`. So I think I'll stick to this unless I misunderstood what you were suggesting

Reply:
Got it. I was just wondering if maybe there was a way to get rid of the `path` prop here, but it seems there isn't.